### DISCUSSION TITLE
isBlurred propterty on CommandMenu

### DISCUSSION THREAD

**Discussion by yuxxeun** [irsyadadl/intentui#226] [Ideas]
It will be so good, if `CommandMenu` has `isBlurred` property like `Aside`, so user can more focused with content on Command Menu

![image](https://github.com/user-attachments/assets/aea27c41-410e-4531-a808-b73a7f032753)

**Comment by irsyadadl** [comment]
I was actually shocked when `isBlurred` wasn't available in the command-menu. I'm so happy if you can give me a PR for that.

**Comment by yuxxeun** [comment]
yes, I did it 🔥🔥

![image](https://github.com/user-attachments/assets/f2a181ca-5e64-49e2-92c6-35d345630673)

 wait i'll make PR

**Comment by yuxxeun** [comment]
Already [added](https://github.com/justdlabs/justd/pull/227) I will close this discussion.

**Reply by irsyadadl** [reply]
Thanks. Merged #228
